src/moonstone-samples/lib/OverlaySample.js

```diff
@@ -0,0 +1,145 @@
+					{kind: Icon, icon: 'star', ontap: 'badgeTapped'},
+					{kind: Icon, icon: 'play', ontap: 'badgeTapped'}
+				]
+			},
+			{kind: GridListImageItem, source: img(300, 300), caption: 'Icon Button', subCaption: 'Transparent BG', ontap: 'itemTapped',
+				overlayShowing: 'spotlight', overlayPosition: 'bottom', overlayAlign: 'right', overlayTransparent: true, overlayComponents: [
+					{kind: Icon, icon: 'play', classes: 'moon-icon-button', style: 'margin: 0; background-color: rgba(0,0,0,0.5)', ontap: 'badgeTapped'}
+				]
+			},
+			{kind: GridListImageItem, source: img(300, 300), caption: 'Centered Icon Button', ontap: 'itemTapped',
+				overlayShowing: 'spotlight', overlayPosition: 'centered', overlayComponents: [
+					{kind: Icon, icon: 'play', small: false, classes: 'moon-icon-button', style: 'margin: 12px;background-color: rgba(0,0,0,0.5)', ontap: 'badgeTapped'}
+				]
+			},
+
+
+			{kind: Divider, content: 'Selection Overlay'},
+			// enyo/DataRepeater adds the `selection-enabled` class when selection is enabled
+			// (appropriately enough). Simulating the same here.
+			{classes: 'selection-enabled', components: [
+				{kind: GridListImageItem, source: img(300, 300), caption: 'Unselected', subCaption: 'Scrimmed', ontap: 'toggleSelected',
+					mixins: [Overlay.Selection], overlayTransparent: false
+				},
+				{kind: GridListImageItem, source: img(300, 300), caption: 'Unselected', subCaption: 'Transparent', ontap: 'toggleSelected',
+					mixins: [Overlay.Selection]
+				},
+				{kind: GridListImageItem, selected: true, source: img(300, 300), caption: 'Selected', subCaption: 'Scrimmed', ontap: 'toggleSelected',
+					mixins: [Overlay.Selection], overlayTransparent: false
+				},
+				{kind: GridListImageItem, selected: true, source: img(300, 300), caption: 'Selected', subCaption: 'Transparent', ontap: 'toggleSelected',
+					mixins: [Overlay.Selection]
+				}
+			]},
+
+
+			{kind: Divider, content: 'Text Overlay'},
+			{kind: GridListImageItem, source: 'http://lorempixel.com/300/300/technics', caption: 'Text Only', ontap: 'itemTapped',
+				mixins: [Overlay.Text], overlayShowing: 'spotlight', overlayTitle: 'Technics', overlaySubtitle: '12'
+			},
+			{kind: GridListImageItem, source: 'http://lorempixel.com/300/300/people', caption: 'Clamped Title', ontap: 'itemTapped',
+				mixins: [Overlay.Text], overlayShowing: 'spotlight', overlayComponents: [
+					// overlayComponents provided by Overlay.Text can be overridden for custom
+					// behavior like clamping the text to 3 lines. If the overlay is not always
+					// visible, clamped must be set to 'always' because the dimensions cannot be
+					// accurately calculated when the ClampedText control isn't showing.
+					{kind: ClampedText, maxLines: 3, clamped: 'always', classes: 'moon-overlay-text-title', content: 'This is probably too long to put in a text overlay but let\'s say you wanted to anyway'},
+					{classes: 'moon-overlay-text-subtitle', content: '3 lines'},
```

Comment:
I approved it before this comma was added. ;) I just figure it's better to catch them now than to have to go back later.